apps/workers/package.json

```diff
@@ -26,6 +26,7 @@
     "metascraper-url": "^5.43.4",
     "ollama": "^0.5.0",
     "openai": "^4.29.0",
+    "pdf-parse": "^1.1.1",
```

Comment:
I'm not a big fan of this package. It seems to be abandoned with the last version being from 5 years ago (https://www.npmjs.com/package/pdf-parse?activeTab=versions). 
Should we maybe use https://github.com/mozilla/pdf.js instead? It even comes with a viewer that we can probably use instead of the `iframe`?

Reply:
I think mozilla/pdf.js is not the right tool for us as it's a PDF viewer for HTML 5.
What do you think of this ?
https://github.com/modesty/pdf2json

Reply:
I don't mind pdf2json. It has a weird api but sounds like it can work 👍

Reply:
It has a weird API indeed. I'll work with it for now until we find another good fit.